<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcClass</class>
 <widget class="QWidget" name="CalcClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>473</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>473</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>352</width>
    <height>473</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
	border:none;
	background-color: rgb(238,236,236);
}

QPushButton
{
	background-color: rgb(249,249,249);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(221,223,221);
}

QPushButton#num0,#num1,#num2,#num3,#num4,#num5,#num6,#num7,#num8,#num9,#num_,#sign
{
	background-color: rgb(255,255,255);
}

QPushButton#num0:hover,#num1:hover,#num2:hover,#num3:hover,#num4:hover,#num5:hover,#num6:hover,#num7:hover,#num8:hover,#num9:hover,#num_:hover,#sign:hover
{
	background-color: rgb(221,223,221);
}

QLineEdit
{
	qproperty-alignment:AlignRight;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="expLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>33</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238,238,236);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnDiv">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="num7">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="num4">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnCE">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnDEL">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>DEL</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="num1">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btnSub">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>19</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="num2">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="num5">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnP">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>xÂ²</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="num0">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="num8">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="num9">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="num6">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnRbk">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnMu">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ã</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btnAdd">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="num_">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnF">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1/x</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnS">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>âx</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="numB">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(41, 134, 255);
hover: rgb(0, 85, 255);</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="num3">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="sign">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnLbk">
       <property name="minimumSize">
        <size>
         <width>86</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>54</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Calc.qrc"/>
 </resources>
 <connections/>
</ui>
